kratos/geometries/nurbs_volume_geometry.h

```diff
@@ -885,6 +885,42 @@ class NurbsVolumeGeometry : public Geometry<typename TContainerPointType::value_
     ///@name Operations
     ///@{
 
+/**
+    * @brief Returns local coordinates in return for a point in physical coordinates.
+    *        This function assumes the initial geometry to be a regular grid. Only linear mapping is applied.
+    *
+    * @param rPointGlobalCoordinates the point to which the
+    *        projection has to be found.
+    * @param rProjectedPointLocalCoordinates the location of the
+    *        projection in local coordinates.
+    * @param Tolerance not used.
+    * @return 1
```

Comment:
Return 1 is used to express that the protection has worked. Maybe extend the comment here that this is easier to understand.